pkg/policy/rule.go

```diff
@@ -209,6 +211,27 @@ func mergeL3(ctx *SearchContext, dir string, ipRules []api.CIDR, resMap *L3Polic
 	return found
 }
 
+func computeResultantCIDRSet(cidrs []api.CIDRRule) []*net.IPNet {
+	var allResultantAllowedCIDRs []*net.IPNet
+	for _, s := range cidrs {
+		_, allowNet, err := net.ParseCIDR(string(s.Cidr))
+		if err != nil {
+			// TODO what to do with error? it should already be valdiated at this point, can we ignore?
```

Comment:
Either document that this is impossible or store the parsed `net.IPNet` in the `api.CIDRRule` as private member so you don't have to parse again on evaluation.

Reply:
Changed the logic here to ignore the error, as all error cases are handled in the validation now. Do you still want me to store the computed `net.IPNet` objects from `api.CIDRRule` in `api.CIDRRule`?